### ISSUE TITLE
Where is the folder smpl_models/smplh/? Do we setup it under the folder lemon_3d?

### ISSUE THREAD

**Issue by MuHe97** [yyvhang/lemon_3d#2]

**Comment by yyvhang** [comment]
Yeah, just create the folder 'smpl_models' under the folder 'lemon_3d'

**Comment by MuHe97** [comment]
Thank you for your support , another question is I did not find the file smpl_neutral_geodesic_dist.npy in [SMPL](https://smpl.is.tue.mpg.de/) (version 1.1.0).  Could you please provied a detail website of the dowload link?

**Comment by MuHe97** [comment]
[Download version 1.1.0 for Python 2.7 (female/male/neutral, 300 shape PCs)](https://download.is.tue.mpg.de/download.php?domain=smpl&sfile=SMPL_python_v.1.1.0.zip) here is the link I download, maybe not this version?

**Comment by yyvhang** [comment]
Sorry for the typo, you can download the smpl_neutral_geodesic_dist.npy from this [RICH](https://rich.is.tue.mpg.de/download.php).
